addons/assignGear/CfgEventHandlers.hpp

```diff
@@ -0,0 +1,32 @@
+class Extended_PreStart_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_FILE(XEH_preStart));
+    };
+};
+class Extended_PreInit_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_FILE(XEH_preInit));
+    };
+};
+
+
+class Extended_Init_EventHandlers {
```

Comment:
Should we add these only if the system is enabled? (CBA lets you run it for all spawned units when adding a new XEH)